src/app/services/match.service.ts

```diff
@@ -21,4 +22,14 @@ export class MatchService {
       .pipe(map(res => res.matches.map(match => new Match(match))))
 
   }
+
+  getConcatinatedSets(sets: Set[]): string {
+    var homeWinSets: number = 0;
```

Comment:
I'd recommend using `let` and `const` rather than var. In this case `let`, as the variable is mutable.

I think it is good practice to use `let` as it block scoped rather than function scoped as `var`. There's also some further weird things going on with `var`. For example that you can redeclare a variable, and that you can access the variable before the initialisation is run.

Some more in-depth explanations of differences: https://stackoverflow.com/a/11444416

Reply:
Fixed now. Thank you for the stackoverflow link!